### DISCUSSION TITLE
How to pin to patch upgrade but allow previous minors

### DISCUSSION THREAD

**Discussion by commentator8** [python-poetry/poetry#7219] [General] [answered]
Hi,

I would like to pin my packages to allow for weekly security updates of patch versions. 

To do so i am using e.g. blah = {version = "~1.3"}. This for a package i am writing and consuming in various microservices and i would like to support various services at various versions that use 1.1, 1.2, 1.3 etc while also having the tilda restrict `poetry update` to patch updates. 

Is there a way to specify that the package can be installed with older versions of blah than 1.3 while still restricting blah to patch updates? e.g. if using 1.2 it should receive 1.2.x updates only, if 1.3 1.3.x only.

Thanks

**Comment by dimbleby** [comment, marked as answer]
no

**Reply by commentator8** [reply]
Thanks, good to know. Does anyone know if this is on any roadmap?

**Reply by dimbleby** [reply]
seems unlikely, PEP440 provides no way to express what you are asking for
